spec/classes/resolv_conf_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'nebula::resolv_conf' do
+  on_supported_os.each do |os, os_facts|
+    context "on #{os}" do
```

Comment:
Ah, nice, tests

Reply:
Yep, just trying to get `firewall` upgraded one step. Intentionally holding back the jump/protocol change for a later pr. I'm not sure which version of `firewall` adds support for the current semantics of `jump`, but I know it's not in `3.6.0`. Just breaking this down into the smallest logical steps.